Source/Fuse.Reactive.Expressions/Tests/UX/UnaryOperator.Basic.ux

```diff
@@ -1,7 +1,14 @@
 <Panel ux:Class="UX.UnaryOperator.Basic">
 	<FuseTest.Data Value=" 'hi' " ux:Name="v"/>
+	<FuseTest.Data StringValue="{= true }" ux:Name="truth"/>
+	<FuseTest.Data StringValue="{= false }" ux:Name="falsehood"/>
 	
 	<FuseTest.DudElement StringValue="{= _unJoin('ab') }" ux:Name="a"/>
 	<FuseTest.DudElement StringValue="{= _unJoin({none}) ?? 'x' }" ux:Name="b"/>
 	<FuseTest.DudElement StringValue="{= _unJoin({v}) }" ux:Name="c"/>
+
+	<FuseTest.DudElement StringValue="{= {truth} }" ux:Name="d"/>
```

Comment:
DudElement also has `BoolValue` to test directly, avoiding the string conversion.

Reply:
@mortoray: Ah, thanks for the pointer!

Reply:
Fixed, much appreciated :)